qa/vagrant/src/test/resources/packaging/scripts/plugin_test_cases.bash

```diff
@@ -93,6 +93,12 @@ fi
     chown -R elasticsearch:elasticsearch "$ESPLUGINS"
 
     install_jvm_example
+    start_elasticsearch_service
```

Comment:
the idea was to install in a custom folder and then check if elasticsearch starts and the plugin is working. do you mean we should install all plugins in custom folder?